### PR TITLE
Add compatibility matrix

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design

/kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Add compatibility matrix for K8s and Golang
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,29 @@ KubeEdge is composed of the following components:
...
+| KubeEdge HEAD       | +               | +               | +               | +               | ✓               | ✓              | -               |
+
+Key:
+* `✓` KubeEdge and the Kubernetes version are exactly compatible.
+* `+` KubeEdge has features or API objects that may not be present in the Kubernetes version.
+* `-` The Kubernetes version has features or API objects that KubeEdge can't use.
+
+### Golang dependency
+
+|                     | Golang 1.10    | Golang 1.11     | Golang 1.12     | Golang 1.13     |
```
Comment (chendave) [inline README.md]: Do we have plan to upgrade golang version to 1.13? 1.13 is formally released.

Comment (fisherxu) [reply README.md]: Yes we will, can upgrade the k8s to 1.16

**Pull request merged**